scss/catppuccin/macchiato.scss

```
@@ -11,6 +11,10 @@ $hover: (
   hover-fg: #8aadf4,
 );
 
+$mark: (
+  search-mark-bg: #f5a97f,
+);
```

Stylelint (17.3.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)